.travis.yml

```diff
@@ -5,15 +5,15 @@ python:
   - 3.4
   - 3.5
 env:
-  - DJANGO="Django>=1.7,<1.8"
   - DJANGO="Django>=1.8,<1.9"
   - DJANGO="Django>=1.9,<1.10"
+  - DJANGO="Django>=1.10,<1.11"
 matrix:
   exclude:
     - python: 3.3
       env: DJANGO="Django>=1.9,<1.10"
-    - python: 3.5
-      env: DJANGO="Django>=1.7,<1.8"
+    - python: 3.3
```

Comment:
That is intentional. Django 1.9+ doesn't support Python 3.3. From the [release docs](https://docs.djangoproject.com/en/dev/releases/1.9/):

> The Django 1.8 series is the last to support Python 3.2 and 3.3.

As such, it must be excluded from the test matrix (as it will fail). So this line is intentionally excluding the combination of Python3.3+Django1.10. Make sense?